### PR TITLE
Methods for running sparcc

### PR DESCRIPTION
**Pull request by wasade**
@EmbrietteH @mortonjt same code as in the gist

### CODE REVIEW

File: scripts/sparcc.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+        # i+1 for upper triangle
+        for col, v in zip(sparcc_z.columns[i+1:], row[i+1:]):
+            if abs(v) >= threshold:
+                fp.write('%s\t%s\t%f\t%d\n' % (idx, col, v, v > 0))
+
+
+def _format_nodes(fp, table):
+    """Format the nodes file
+    Paramters
```
Comment (mortonjt) [inline scripts/sparcc.py]: Paramters -> Parameters

File: scripts/sparcc.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    table = biom.load_table(table)
+    _, _, tmp_md = next(table.iter(axis='observation'))
+    header.extend(sorted(tmp_md.keys()))
+
+    fp.write("\t".join(header))
+    fp.write('\n')
+
+    for values, id_, md in table.iter(axis='observation'):
+        line = [str(id_), str(values.sum())]
```
Comment (mortonjt) [inline scripts/sparcc.py]: Why is there a sum happening here?  Is this a requirement for the QIIME formatted table?

**Comment by mortonjt** [comment]
Couple of minor comments.  I think it is good to go.

File: scripts/sparcc.py (hunk 2 above)
Comment (wasade) [inline scripts/sparcc.py]: This was being performed by the code I based this off. @sowe9385, can you comment?

Comment (sowe9385) [inline scripts/sparcc.py]: Adding a feature sum into the node table, so that users can color the nodes by abundance in Cytoscape.

**Comment by jwdebelius** [comment]
@wasade and @mortonjt : could you please trouble shoot the travis build?

**Comment by wasade** [comment]
@jwdebelius i believe its addressed

**Pull request merged**